src/components/Room/Room.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import HomePage from '../HomePage/index';
+import { withRouter } from 'react-router';
+import { updateSongs } from '../SongQueue/SQueueActions';
+import io from 'socket.io-client';
+const socket = io('http://localhost:3000');
```

Comment:
Remove localhost from this line
```suggestion
const socket = io();
```